packages/styletron-standard/src/__tests__/core.node.js

```diff
@@ -1,81 +0,0 @@
-// @flow
```

Comment:
this was removed to avoid circular dependency(which typescript project references does not allow), also what is here is already tested in `styletron-engine-monolithic` and `styletron-engine-atomic`